build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "Useage: ./build.sh LANGUAGE"
+    echo
+    echo "Must specify language. Please choose one of:"
+    echo "  binary"
+    echo "  go"
+    echo "  java"
+    echo "  nodejs"
+    echo "  dotnet-core"
+    echo "  php"
+    echo "  python"
+    echo "  ruby"
+    echo "  staticfile"
+    exit;
+fi
+
+LANGUAGE=$1
+GO_VERSION="1.7"
+DIEGO_VERSION=$(curl -s -L https://api.github.com/repos/cloudfoundry/diego-release/releases/latest | jq -r .tag_name)
```

Comment:
Might be worth mentioning `jq` as a prerequisite in the README?  I didn't have `jq` installed on my system, but `brew install jq` fixed things for me.